hooks/lib/hooks/common.py

```diff
@@ -15,7 +15,7 @@
 
 PUBLIC_DOMAIN_PREFIX = "%PUBLIC_DOMAIN%://"
 CLUSTER_DOMAIN_PREFIX = "%CLUSTER_DOMAIN%://"
-
+MODULE_NAME = "virtualization"
```

Comment:
this is a library. We don't have to hard code our module in this.
We already have this var https://github.com/deckhouse/virtualization/blob/683bb70318210e89755de237d813a9720911395d/hooks/common.py#L25